hack/local-up-cluster.sh

```diff
@@ -221,6 +221,7 @@ function start_apiserver {
       --port="${API_PORT}" \
       --etcd_servers="http://127.0.0.1:4001" \
       --service-cluster-ip-range="10.0.0.0/24" \
+      --runtime-config=experimental/v1=true \
```

Comment:
This probably slipped in by accident, but experimental should be disabled by default.